module.nix

```diff
@@ -30,7 +28,7 @@ in
       priority = 50;
     };
 
-    environment.systemPackages = [ harmonia ];
+    environment.systemPackages = [ pkgs.harmonia ];
```

Comment:
This is using harmonia from nixpkgs instead. If users use this flake would they not expect to use harmonia from this repository instead?

Reply:
I have a follow on all harmonia inputs to reduce the amount of evaluated nixpkgs from 10+ to just a few https://gitea.c3d2.de/c3d2/nix-config/src/branch/master/flake.nix#L92-L99

To fix the build failure caused by a to old rust, I am then using an overlay https://gitea.c3d2.de/c3d2/nix-config/src/branch/master/flake.nix#L439-L443

That did not work before.